### PR TITLE
gayazo support and jpeg support for imgur

### PR DESCRIPTION
**Pull request by zoranzaric**

### CODE REVIEW

File: src/collector/views.py [hunk 1]
```diff
@@ -53,6 +53,24 @@ def findImage(url):
...
 		turl = "https://i.imgur.com" + urlparse(url).path + ".jpg"
 		res = requests.head(turl)
+		if res.status_code == 200:
+			return turl
+		turl = "https://i.imgur.com" + urlparse(url).path + ".jpeg"
+		res = requests.head(turl)
+		if res.status_code == 200:
+			return turl
+		return None
+	elif host in ["gayazo.com"]:
```
Comment (fishfacemcgee) [inline src/collector/views.py:63]: Shouldn't this be gyazo? https://github.com/Corridors-of-Time-Transcription/puzzlepieces/issues/55

**Comment by fishfacemcgee** [comment]
@thorsteneb Pretty sure the URL being parsed in this isn't correct, FWIW.
image

File: src/collector/views.py (hunk 1 above)
Comment (zoranzaric) [reply src/collector/views.py:63]: absolutely correct...

**Comment by TrueSkrillor** [comment]
Already noticed, will be fixed soon. It doesn't break anything so everything's fine here

**Comment by zoranzaric** [comment]
i pushed a fix to my branch

**Pull request merged**